src/main.ts

```diff
@@ -13,13 +13,11 @@ const vueLifecycles = singleSpaVue({
     appOptions: {
         render() {
             return h(App, {
-                props: {
-                    // single-spa props are available on the "this" object. Forward them to your component as needed.
-                    // https://single-spa.js.org/docs/building-applications#lifecyle-props
-                    name: this.name,
-                    mountParcel: this.mountParcel,
-                    singleSpa: this.singleSpa
-                }
+                // single-spa props are available on the "this" object. Forward them to your component as needed.
+                // https://single-spa.js.org/docs/building-applications#lifecyle-props
+                // name: this.name,
```

Comment:
This is to eliminate console warnings, since the props here aren't defined in App.vue